### PR TITLE
add sendToChat tests

### PR DESCRIPTION
**Pull request by adbenitez**
requires https://github.com/deltachat/deltachat-android/pull/2571 when testing on Android and https://github.com/deltachat/deltachat-ios/pull/1886 when testing on iOS

### CODE REVIEW

File: webxdc.js [hunk 1]
```diff
@@ -51,6 +51,9 @@ window.webxdc = (() => {
             console.log('[Webxdc] description="' + description + '", ' + JSON.stringify(_update));
             updateListener(_update);
         },
+        sendToChat: (content) => {
```
Comment (Simon-Laux) [inline webxdc.js:54]: you can copy what I did in the simulator for the calendar app https://github.com/webxdc/calendar/pull/11/files#diff-4dc52a56e760eb8eebd14247c25417f6a9a8a0d7cd905787a088b169d6ef197f

Comment (adbenitez) [reply webxdc.js:54]: I think the right place to update is in the [hello](https://github.com/webxdc/hello) repo, that is the template where [webxdc.js](https://github.com/webxdc/hello/blob/master/webxdc.js) mini-emulator and [webxdc.d.ts](https://github.com/webxdc/hello/blob/master/webxdc.d.ts) are maintained, then just copy+paste the files in the other places that need it

Comment (adbenitez) [reply webxdc.js:54]: created an issue: https://github.com/webxdc/hello/issues/45

probably the other advanced emulator also needs updating, not sure if it is a matter to also copying the webxdc.js file from the hello project

**Review by r10s — APPROVED** [review]
lgtm, also works as expected.

i added a commit that shows rejected promise.

i do not think, non-updated webxdc.js should block merging, it is not really needed for usage in the apps for testing, it is not good if too many pr with minor things wait on each other. 
i'll add a note to hello that we should update webxdc.js here later on

**Pull request merged**